CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Pipeline has been re-implemented in [Nextflow DSL2](https://www.nextflow.io/docs
 - [#90](https://github.com/nf-core/metapep/pull/90) - Removed script headers and unused imports
 - [#91](https://github.com/nf-core/metapep/pull/91) - Added new parameter to allow all input peptide lengths to be predicted, and also necessary checks to circumvent errors during predictions
 - [#94](https://github.com/nf-core/metapep/pull/94) - Updated workflow overview figure
+- [#96](https://github.com/nf-core/metapep/pull/96) - Rework of chunk sizes and adjusting the parameters to feasible job size and number for large datasets
```

Comment:
```suggestion
- [#96](https://github.com/nf-core/metapep/pull/96) - Rework of chunk sizes and adjusting the parameters to optimise the number and size of jobs for large datasets
```

Maybe mention also which parameters changed or were added? 
Not exactly sure what you mean with the last half sentence, number refers to "feasible job"?

Reply:
```suggestion
- [#96](https://github.com/nf-core/metapep/pull/96) - Rework of chunk sizes and adjusting and creating the corresponding parameters (`--prediction_chunk_size`, `--pred_chunk_size_scaling`, `--downstream_chunk_size`, `--pred_buffer_files`) to optimise the number and size of jobs for large datasets
```

Reply:
@skrakau you mean something like this?